BMICalculator/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace BMICalculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        	// To be able to use "." instead of "," as decimal separator
+            System.Globalization.CultureInfo customCulture = (System.Globalization.CultureInfo)System.Threading.Thread.CurrentThread.CurrentCulture.Clone();
+            customCulture.NumberFormat.NumberDecimalSeparator = ".";
+            System.Threading.Thread.CurrentThread.CurrentCulture = customCulture;
+
+            for (int i = 0; i < 2; i++)
+            {
```

Comment:
A single iteration could be a function call and just call that function twice. Loop is a little overkill.

Reply:
Thank you for the review. I've updated the code.